xml/art-obs-beginners-guide.xml

```diff
@@ -1038,26 +1114,34 @@ Patch0:         &gitproject;_main.diff</screen>
     osc submitreq [OPTIONS] DESTPRJ [DESTPKG]
     osc submitreq [OPTIONS] SOURCEPRJ SOURCEPKG DESTPRJ [DESTPKG]
     -->
-    <screen>&prompt.user2;<command>osc</command> submitpac &obshome1;</screen>
+    <screen>&prompt.user2;<command>osc</command> submitpac</screen>
     <para>
-     &obsa; knows where the original project is located, so you do not have
-     to add the project name here.
+     &obsa; usually knows where the original project is located, so you do
```

Comment:
Added additional sentence to clarify what happens when you use "osc submitreq" only:

```
Usually, &obsa; you do not have to add the project or package names here.
By default, <command>submitreq</command> without any options submits back
to the package where you branched from.
```